mongo/README.md

```diff
@@ -145,7 +145,7 @@ For containerized environments, see the [Autodiscovery Integration Templates][8]
 | -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
 | `<INTEGRATION_NAME>` | `mongo`                                                                                                                                   |
 | `<INIT_CONFIG>`      | blank or `{}`                                                                                                                             |
-| `<INSTANCE_CONFIG>`  | `{"hosts": ["%%hosts%%:%%port%%], "username": "datadog", "password : "<UNIQUEPASSWORD>", "database": "<DATABASE>", "replica_check": true}` |
+| `<INSTANCE_CONFIG>`  | `{"host": ["%%hosts%%:%%port%%"], "username": "datadog", "password : "<UNIQUEPASSWORD>", "database": "<DATABASE>", "replica_check": true}` |
```

Comment:
```suggestion
| `<INSTANCE_CONFIG>`  | `{"hosts": ["%%host%%:%%port%%"], "username": "datadog", "password : "<UNIQUEPASSWORD>", "database": "<DATABASE>", "replica_check": true}` |
```
I think this is the correct fix, isn't it? The config field is correctly spelled (check https://github.com/DataDog/integrations-core/blob/master/mongo/datadog_checks/mongo/data/conf.yaml.example#L29), but `%%hosts%%` is not valid AFAIK

Reply:
The customer followed the sample [Redis configuration](https://docs.datadoghq.com/agent/kubernetes/integrations/?tab=file#datadog-redis-integration) where is no s for both host so the syntax that works is `{"host": ["%%host%%:%%port%%"]`

Reply:
Yes but this is mongo here

Reply:
I'll ask the customer to provide the configuration that works in their environment and update you again